src/handlers/yarn.js

```diff
@@ -0,0 +1,51 @@
+import ora from 'ora';
+import fetch from 'isomorphic-fetch';
+import { exec, log } from '../util';
+
+let originalVersion;
+
+async function afterAll() {
+  await exec(`yarn add ${originalVersion} --no-lockfile`);
+}
+
+async function getCandidates({ module }) {
+  console.log(`Fetch version data from registry.npmjs.org`);
+
+  const response = await fetch(`https://registry.npmjs.org/${module}`);
+  const data = await response.json();
+
+  return Object.keys(data.versions || {});
+}
+
+async function testCandidate(candidate, attempt, { module }) {
+  if (!originalVersion) {
```

Comment:
I couldn't put this in `beforeAll` because I need the module name.

Reply:
We can of course also update the `beforeAll` call to include `args`? `beforeAll(args)`. What do you think?

Reply:
Yeah, I also considered that. I'm happy to do that if you prefer.